nilearn/_utils/fixes/__init__.py

```diff
@@ -10,11 +10,8 @@
     from sklearn.feature_selection import f_regression
 
 try:
-    if LooseVersion(sklearn.__version__) >= LooseVersion('0.18'):
-        from sklearn.model_selection import check_cv
     # 0.18 > scikit-learn >= 0.16
-    else:
-        from sklearn.cross_validation import check_cv
+    from sklearn.cross_validation import check_cv
```

Comment:
I don't understand. That's the wrong change: it is importing the deprecated check_cv in 0.18. We shouldn't use the sklearn.cross_validation module in sklearn > 0.18, we should use sklearn.model_selection.